diff --git a/node_modules/expo-device/ios/UIDevice.swift b/node_modules/expo-device/ios/UIDevice.swift
index f1968fa..a2ff3c9 100644
--- a/node_modules/expo-device/ios/UIDevice.swift
+++ b/node_modules/expo-device/ios/UIDevice.swift
@@ -1,4 +1,5 @@
 import UIKit
+import TargetConditionals // ✅ Añadido
 
 struct ExpoDeviceType {
   let modelName: String
@@ -172,8 +173,16 @@ public extension UIDevice {
   // swiftlint:enable function_body_length closure_body_length
 
   // Credit: https://github.com/developerinsider/isJailBroken/blob/master/IsJailBroken/Extension/UIDevice%2BJailBroken.swift
-  var isSimulator: Bool {
+  /*var isSimulator: Bool {
     return TARGET_OS_SIMULATOR != 0
+  }*/
+
+       var isSimulator: Bool {
+      #if targetEnvironment(simulator)
+            return true
+    #else
+          return false
+    #endif
   }
 
   var isJailbroken: Bool {
